endpoints-frameworks-v2/backend/pom.xml

```diff
@@ -0,0 +1,119 @@
+<!--
+ Copyright 2016 Google Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+
+    <groupId>com.example.echo</groupId>
+    <artifactId>echo</artifactId>
+
+    <parent>
+        <artifactId>doc-samples</artifactId>
+        <groupId>com.google.cloud</groupId>
+        <version>1.0.0</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <properties>
+        <appengine.app.version>1</appengine.app.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+
+        <appengine.plugin.version>1.9.40</appengine.plugin.version>
+
+        <endpoints.management.version>0.0.18-SNAPSHOT</endpoints.management.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>internal-endpoints-testing</id>
+            <name>Internal Endpoints Testing</name>
+            <url>http://104.197.230.53:8081/nexus/content/repositories/snapshots/</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <!-- Compile/runtime dependencies -->
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.endpoints</groupId>
+            <artifactId>endpoints-framework</artifactId>
```

Comment:
We have some ideas on how to bring the number down from 3, but it's still in discussion.